build.sbt

```diff
@@ -110,6 +113,11 @@ lazy val rawls = project.in(file("."))
     testOptions in Test ++= Seq(Tests.Filter(s => !isIntegrationTest(s))),
     testOptions in IntegrationTest := Seq(Tests.Filter(s => isIntegrationTest(s)))
   )
+  .settings(
+    buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion, "swaggerUIVersion" -> swaggerUIV),
```

Comment:
Do we need these other keys?  And why not name the val `swaggerUIVersion` to begin with?